libglusterfs/src/inode.c

```diff
@@ -1682,26 +1684,22 @@ static void
 __inode_table_init_root(inode_table_t *table)
 {
     inode_t *root = NULL;
-    struct iatt iatt = {
-        0,
-    };
-
-    if (!table)
-        return;
+    static uuid_t root_gfid = {0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1};
 
     root = inode_create(table);
 
     list_add(&root->list, &table->lru);
     table->lru_size++;
     root->in_lru_list = _gf_true;
 
-    iatt.ia_gfid[15] = 1;
-    iatt.ia_ino = 1;
-    iatt.ia_type = IA_IFDIR;
+    gf_uuid_copy(root->gfid, root_gfid);
```

Comment:
Why did we ignore ia_ino assignment?

Reply:
we removed the need for iatt itself completely!

Reply:
> Why did we ignore ia_ino assignment?

I did not see where it was actually used.